simbad/lattice/latticesearch.py

```diff
@@ -324,15 +331,18 @@ def download_results(self, destination):
 
         to_del = []
         for count, result in enumerate(self.results):
-
-            content = get_pdb_content(result.pdb_code)
+            content = simbad.util.pdb_util.get_pdb_content(result.pdb_code)
             if content is None:
```

Comment:
Little side-note, don't check for inequality first. Swap these around and write something like:

```python
if content:
   f_tmp_out = tempfile.Nam ...
else:
    logger.warning(...)
    to_del.append(count)